### ISSUE TITLE
Removal Candidate

### ISSUE THREAD

**Issue by Adurol** [KaOS-Community-Packages/iscan#3]
This Package will be removed in 7 days from KCP.

Reason: **Unmaintained/Outdated**
This PKGBUILD is most likely a blind copy from CCR or AUR. But it's so outdated, I couldn't find the origin anymore.

Please see also: [Using KaOS Community Packages](https://kaosx.us/phpBB3/viewtopic.php?f=12&t=145)

**Comment by demmm** [comment]
Nope, original was mine from around 2010 (ccr might have copied it by now :)  )
But it was added on a request, with the premise the requester would maintain, hasn't happened, so seems no users for this package anymore.

**Comment by Adurol** [comment]
Thanks for clarification.
If no user of a Brother scanner hops on to maintain it, it will get removed.
